### DISCUSSION TITLE
Version about the first formal Apache release

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks-controller#191] [General]
Hi community,

I'm going to call the first formal release after entering the Apache organization
and would like to propose it as 1.0.0. What do you think about this? @apache/kvrocks-committers

**Comment by jihuayu** [comment]
Does 1.0.0 include UI and embedded storage?

**Reply by git-hulk** [reply]
Would not, only contains current merged commits.

**Comment by git-hulk** [comment]
Will release it as 1.0.0 since there's no objection for now.
